asdf.py

```diff
@@ -127,10 +127,11 @@ def _run_command(self, command, message=None, error_message=None, **kwargs):
 
         if self._system_sh_is_dash():
             # dash doesn't overwrite $SHELL so lets try to use that.
-            shell = getenv("SHELL")
+            shell = getenv("SHELL", default="/usr/bin/bash")
             self._log.debug(f"dash detected, attempting to use user shell {shell}")
-            command = f'{shell} -c "{command}"'
+            command = '%s -c "%s"' % (shell, command)
```

Comment:
Please, don't use `%s`

Reply:
Oh, I was using fstrings but I changed it to be more backwards compatible. Should I change back to fstrings or do you prefer something else?

Reply:
`.format` should be the best in this case! 👍